be_wmg_aws/lift_studies/utils/data_utils.py

```diff
@@ -12,7 +12,7 @@ class LiftS3Handler:
     """
 
     def __init__(self):
-        self.region = os.environ['AWS_REGION']
+        self.region = os.environ["AWS_REGION"] # noqa: F821
```

Comment:
Sorry. Linter requires 2 spaces before the comment, please adjust.
self.region = os.environ["AWS_REGION"] (make sure there are 2 spaces here) # noqa: F821